fuzzers/structure_aware/baby_fuzzer_custom_input/src/input.rs

```diff
@@ -25,15 +23,14 @@ use serde::{Deserialize, Serialize};
 pub struct CustomInput {
     pub byte_array: Vec<u8>,
     pub optional_byte_array: Option<Vec<u8>>,
+    pub num: i16,
     pub boolean: bool,
 }
 
 /// Hash-based implementation
 impl Input for CustomInput {
     fn generate_name(&self, _id: Option<CorpusId>) -> String {
-        let mut hasher = DefaultHasher::new();
-        self.hash(&mut hasher);
-        format!("{:016x}", hasher.finish())
+        format!("{:016x}", RandomState::with_seed(0).hash_one(self))
```

Comment:
libafl has hasher_std for this https://docs.rs/libafl_bolts/latest/libafl_bolts/fn.hasher_std.html